packages/lib-classifier/src/plugins/tasks/DrawingTask/components/helpers/getIcon.js

```diff
@@ -43,7 +43,7 @@ const icons = {
 }
 
 function getIcon (icon) {
-  return icons[icon] || null
+  return icons[icon] || (() => null)
```

Comment:
I don't understand the purpose of this change.

Reply:
The original code will crash the classifier if there's no icon, since `null` isn't a React component.